CHANGELOG.md

```diff
@@ -34,6 +37,15 @@ See related PR: [moby/moby#35812](https://github.com/moby/moby/pull/35812)
 - Honor `DOCKER_RAMDISK` with containerd 1.0 [moby/moby#35957](https://github.com/moby/moby/pull/35957)
 - Update runc to fix hang during start and exec [moby/moby#36097](https://github.com/moby/moby/pull/36097) 
 - Windows: Vendor of Microsoft/hcsshim @v.0.6.8 partial fix for import layer failing [moby/moby#35924](https://github.com/moby/moby/pull/35924)
+* Do not make graphdriver homes private mounts [moby/moby#36047](https://github.com/moby/moby/pull/36047)
+* Use rslave propogation for mounts from daemon root [moby/moby#36055](https://github.com/moby/moby/pull/36055)
+* Set daemon root to use shared propagation(https://github.com/moby/moby/pull/36096)[moby/moby#36096](https://github.com/moby/moby/pull/36096)
+* Validate Mount-specs on container start to prevent creating missing host-paths [moby/moby#35833](https://github.com/moby/moby/pull/35833)
```

Comment:
Consider rewording to
"Validate that mounted paths exist when container is started, not just during creation"